simulators/portal-interop/src/main.rs

```diff
@@ -175,10 +185,60 @@ dyn_async! {
                     client_b: &(*client_b).clone(),
                 }
             ).await;
+
+            // Test find content immediate return
+            test.run(TwoClientTestSpec {
+                    name: format!("FIND_CONTENT {} --> {}", client_a.name, client_b.name),
+                    description: "find content: peer immediately returns locally available content".to_string(),
+                    always_run: false,
+                    run: test_find_content_immediate_return,
+                    client_a: &(*client_a).clone(),
+                    client_b: &(*client_b).clone(),
+                }
+            ).await;
         }
    }
 }
 
+dyn_async! {
+    // test that a node will return content via FINDCONTENT that it has stored locally
+    async fn test_find_content_immediate_return<'a> (test: &'a mut Test, client_a: Client, client_b: Client) {
+        let header_with_proof_key: HistoryContentKey = serde_json::from_value(json!(HEADER_WITH_PROOF_KEY)).unwrap();
+        let header_with_proof_value: HistoryContentValue = serde_json::from_value(json!(HEADER_WITH_PROOF_VALUE)).unwrap();
+
+        match client_b.rpc.store(header_with_proof_key.clone(), header_with_proof_value.clone()).await {
+            Ok(result) => if !result {
+                test.fatal("Unable to store header with proof for Block Body verification");
```

Comment:
I think we need to change the error message here and below because we are not actually storing the header for verification of a block body.